src/ripple/app/tx/impl/SetRegularKey.cpp

```diff
@@ -74,7 +74,10 @@ SetRegularKey::doApply ()
     auto const sle = view().peek(
         keylet::account(account_));
 
-    if (mFeeDue == zero)
+	auto const feeDue = calculateFee(ctx_.app, ctx_.baseFee,
+		view().fees(), view().flags());
+
+	if (feeDue == zero)
```

Comment:
Formatting is off here (not indented enough).

We don't need the extra variable `feeDue`. Also, since `XRPAmount` is convertable to a bool, we could just:`if (!calculateFee(...`. The extra variable and ` == zero` don't really bother me; your call.

Reply:
Fixed, I think.